website/addons/github/static/githubFangornConfig.js

```diff
@@ -285,13 +291,27 @@ function gotoFile (item) {
         window.open(fileurl, '_self');
     }
 }
+function getParameterByName(name) {
```

Comment:
You can use `$osf.urlParams()` to get the url parameters instead of adding this new function.